raiden/ui/cli.py

```diff
@@ -628,10 +634,12 @@ def run(ctx: Context, **kwargs: Any) -> None:
         )
         sys.exit(1)
     finally:
-        if profiler is not None:
-            profiler.stop()
+        if switch_monitor is not None:
```

Comment:
This is a bit problematic, the traces have to be stopped in the reverse order of their instantiation.